src/core/configuration/DefaultConfig.ts

```
@@ -521,6 +524,14 @@ export class DefaultConfig implements Config {
     return base;
   }
 
+  private startingTilesFor(player: Player | PlayerView): Array<TileRef> | undefined {
+    if (this._gameConfig.startingOwnedTiles == undefined) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)